content/general/concepts/number/number.md

```diff
@@ -0,0 +1,15 @@
+---
+Title: 'Numbers'
+Description: 'The numbers data type.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Data Types'
+  - 'Types'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Numbers include `-1`, `0`, `1`, and so on. Depending on the language, numbers with decimals are of a different type. When computing numbers such they trade-off accuracy for range. Witch means they trade-off how acute it is with how many numbers it can compute. Languages such as Javascript and Python do this. This is why when you do this `.1+.2` it would equal `0.30000000000000004`. But if you are adding whole numbers it works all the time. But you need to keep in mind the integer limit. It is the number where the compiler won't understand the numbers anymore.
```

Comment:
```suggestion
Numbers include `-1`, `0`, `1`, and so on. Depending on the language, numbers with decimals are of a different type. When computing numbers such they trade-off accuracy for range. Witch means they trade-off how acute it is with how many numbers it can compute.

## Arithmetic Accuracy 

Languages such as Javascript and Python yield interesting results when arithmetic operations are performed on numeric values. For example, `.1+.2` would evaluate tol `0.30000000000000004`, whereas adding whole numbers wouldn't do this. But you need to keep in mind the integer limit. It is the number where the compiler won't understand the numbers anymore.
```

Some other **thoughts**:

> Numbers include `-1`, `0`, `1`, and so on

I think this should be turned into a list that includes things like:

* Integers, which are whole numbers that include values like `-1`, `0`, and `1`.

> But you need to keep in mind the integer limit. It is the number where the compiler won't understand the numbers anymore.

For the first sentence, could we unpack that a bit more? "[L]imit" of what? As for the second sentence, what does this mean?